test/bench/granular/boolean.bench.ts

```diff
@@ -0,0 +1,49 @@
+import { Suite } from 'dbx-js-tools/packages/bson-bench';
+import {
+  getTestDocs,
+  runSuiteAndWriteResults,
+  BOOL,
+  ITERATIONS,
+  WARMUP,
+  LIBRARY_SPEC
+} from './common';
+
+const OPTIONS = {
+  serialize: { checkKeys: true, ignoreUndefined: false },
+  deserialize: {
+    promoteValues: true,
+    index: 0
+  }
+};
+
+async function main() {
+  const suite = new Suite('Boolean');
+  const testDocs = await getTestDocs('boolean');
+  // deserialize
+  for (const documentPath of testDocs) {
+    for (const promoteValues of BOOL) {
+      suite.task({
+        documentPath,
+        library: LIBRARY_SPEC,
+        iterations: ITERATIONS,
+        warmup: WARMUP,
+        operation: 'deserialize',
+        options: { ...OPTIONS['deserialize'], promoteValues }
```

Comment:
```suggestion
        options: { ...OPTIONS.deserialize, promoteValues }
```

I see this in a number of places, but I won't comment on more, any reason for the change in syntax?